scripts/reanalyze_sample_games.py

```
@@ -64,30 +73,30 @@ async def reanalyze_sample_games():
                 else:
-                    print(f"   ⚠️  Accuracy decreased by {abs(improvement):.1f}%")
+                    print(f"   Accuracy decreased by {abs(improvement):.1f}%")
             else:
-                print(f"   ❌ Re-analysis failed")
+                print(f"   Re-analysis failed")
         
-        print(f"\n📋 RE-ANALYSIS COMPLETE!")
-        print(f"   • These games now use the new accuracy settings")
-        print(f"   • You can see the improved accuracy in your app")
-        print(f"   • To re-analyze more games, run this script again")
+        print(f"\nRE-ANALYSIS COMPLETE!")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)